addons/google_calendar/models/google_calendar.py

```diff
@@ -352,116 +351,111 @@ def get_one_event_synchro(self, cr, uid, google_id, context=None):
                 if type == "write":
                     for oe_attendee in event['attendee_ids']:
                         if oe_attendee.email == google_attendee['email']:
-                            calendar_attendee_obj.write(cr, uid, [oe_attendee.id], {'state': google_attendee['responseStatus']}, context=context)
+                            CalendarAttendee.browse(oe_attendee.id).write({'state': google_attendee['responseStatus']})
                             google_attendee['found'] = True
                             continue
 
                 if google_attendee.get('found'):
                     continue
 
-                attendee_id = res_partner_obj.search(cr, uid, [('email', '=', google_attendee['email'])], context=context)
-                if not attendee_id:
+                attendees = ResPartner.search([('email', '=', google_attendee['email'])])
+                if not attendees:
                     data = {
                         'email': google_attendee['email'],
                         'customer': False,
                         'name': google_attendee.get("displayName", False) or google_attendee['email']
                     }
-                    attendee_id = [res_partner_obj.create(cr, uid, data, context=context)]
-                attendee = res_partner_obj.read(cr, uid, attendee_id[0], ['email'], context=context)
+                    attendees = [ResPartner.create(data)]
+                #attendee = res_partner_obj.browse(attendees[0].id).read(['email'])[0]
+                attendee = attendees[0].read(['email'])[0]
                 partner_record.append((4, attendee.get('id')))
                 attendee['partner_id'] = attendee.pop('id')
                 attendee['state'] = google_attendee['responseStatus']
                 attendee_record.append((0, 0, attendee))
         for google_alarm in single_event_dict.get('reminders', {}).get('overrides', []):
-            alarm_id = calendar_alarm_obj.search(
-                cr,
-                uid,
+            alarms = CalendarAlarm.search(
                 [
                     ('type', '=', google_alarm['method'] if google_alarm['method'] == 'email' else 'notification'),
                     ('duration_minutes', '=', google_alarm['minutes'])
                 ],
-                context=context
             )
-            if not alarm_id:
+            if not alarms:
                 data = {
                     'type': google_alarm['method'] if google_alarm['method'] == 'email' else 'notification',
                     'duration': google_alarm['minutes'],
                     'interval': 'minutes',
                     'name': "%s minutes - %s" % (google_alarm['minutes'], google_alarm['method'])
                 }
-                alarm_id = [calendar_alarm_obj.create(cr, uid, data, context=context)]
-            alarm_record.append(alarm_id[0])
+                alarms = [CalendarAlarm.create(data)]
+            alarm_record.append(alarms[0].id)
```

Comment:
@rga-odoo I don't think so we should add two alarms with same duration and type, does it make sense to have two alarms with same duration and type ?